source/dicom/UID.py

```diff
@@ -133,12 +133,41 @@ def __hash__(self):
 ImplicitVRLittleEndian = UID('1.2.840.10008.1.2')
 DeflatedExplicitVRLittleEndian = UID('1.2.840.10008.1.2.1.99')
 ExplicitVRBigEndian = UID('1.2.840.10008.1.2.2')
+JPEGBaseLineLossy8bit = UID('1.2.840.10008.1.2.4.50')
+JPEGBaseLineLossy12bit = UID('1.2.840.10008.1.2.4.51')
+JPEGLossless = UID('1.2.840.10008.1.2.4.70')
+JPEGLSLossless = UID('1.2.840.10008.1.2.4.80')
+JPEGLSLossy = UID('1.2.840.10008.1.2.4.81')
+JPEG2000Lossless = UID('1.2.840.10008.1.2.4.90')
+JPEG2000Lossy = UID('1.2.840.10008.1.2.4.91')
 
 NotCompressedPixelTransferSyntaxes = [ExplicitVRLittleEndian,
                                       ImplicitVRLittleEndian,
                                       DeflatedExplicitVRLittleEndian,
                                       ExplicitVRBigEndian]
 
+UncompressedPixelTransferSyntaxes = [ExplicitVRLittleEndian,
```

Comment:
Why do you now have `UncompressedPixelTransferSyntaxes` and `NotCompressedPixelTransferSyntaxes`?